test/lambda_overload_test.cpp

```diff
@@ -112,6 +115,47 @@ void test_match_overloads_capture()
     std::cout << "Got " << ok << " ok, " << err << " err" << std::endl;
 }
 
+struct MovableOnly
+{
+    MovableOnly() = default;
+    MovableOnly(MovableOnly&&) = default;
+};
+
+struct MovableCopyable
+{
+    MovableCopyable() = default;
+    MovableCopyable(MovableCopyable&&) = default;
+    MovableCopyable(const MovableCopyable&) = default;
+};
```

Comment:
Thanks @daniel-j-h! We're not actually using the assignment operator in the test, but it's important to an object tagged as movable and/or copyable. Pull request has been updated.